README.md

```diff
@@ -73,5 +70,5 @@ In the file browser, you will now see a folder for the repository that was clone
 
 ## Open the Notebook
 At this point you should double-click on the `rhods-fraud-detection` folder in
-the file explorer, and then double-click on the `Notebook.ipynb`
+the file explorer, and then double-click on the [Notebook.ipynb](./Notebook.ipynb)
 notebook file. Begin to follow the instructions in that notebook.
```

Comment:
Can we attach a screen shot for JH nb with running one of the cells

Reply:
i will do that